composer.json

```diff
@@ -14,14 +14,19 @@
     ],
 
     "require": {
-        "php"                           : ">=5.5",
-        "sensio/framework-extra-bundle" : "^3.0.0",
-        "symfony/translation"           : "^3.0.2|^2.3.0",
-        "symfony/yaml"                  : "^3.0.2|^2.3.0"
+        "php"                          : "^7.0|^5.6",
+        "symfony/config"               : "^3.0|^2.3",
+        "symfony/dependency-injection" : "^3.0|^2.3",
+        "symfony/framework-bundle"     : "^3.0|^2.7",
```

Comment:
Not necessarily, the framework-bundle can also require components of version 2.3 and does not force people to also use the newer version of their components. Sadly I cannot lower this to 2.3 because of the kernel test case and 2.5/2.6 are unsupported (added in 2.5), see symfony/symfony#17749.